tdrs-backend/tdpservice/parsers/validators/category1.py

```diff
@@ -0,0 +1,215 @@
+"""Overloads and custom validators for category 1 (preparsing)."""
+
+from tdpservice.parsers.models import ParserErrorCategoryChoices
+from tdpservice.parsers.util import fiscal_to_calendar, year_month_to_year_quarter
+from .base import ValidatorFunctions
+from .util import ValidationErrorArgs, make_validator, _is_all_zeros, _is_empty, value_is_empty
+
+
+def format_error_context(eargs: ValidationErrorArgs):
+    """Format the error message for consistency across cat1 validators."""
+    return f'{eargs.row_schema.record_type} Item {eargs.item_num} ({eargs.friendly_name}):'
+
+
+class PreparsingValidators():
```

Comment:
this is simply using Base static function not overloading class methods. Why you are not inheriting from base class?

Reply:
updated the validators modules to just export the functions directly, as discussed in OH